ldk/go/keyboardService.go

```diff
@@ -20,11 +20,12 @@ func (h Hotkey) Match(v Hotkey) bool {
 		return false
 	}
 	// You can't do a direct bitmask as Alt means Altleft or AltRight.  So checking all combinations is the only safe way to actually compare
-	l, r := h.Modifiers, v.Modifiers
-	return (l.AltLeft() == r.AltLeft() || l.Alt() == r.Alt()) && (l.AltRight() == r.AltRight() || l.Alt() == r.Alt()) &&
-		(l.ControlLeft() == r.ControlLeft()) || l.Control() == r.Control() && (l.ControlRight() == r.ControlRight() || l.Control() == r.Control()) &&
-		(l.MetaLeft() == r.MetaLeft() || l.Meta() == r.Meta()) && (l.MetaRight() == r.MetaRight() || l.Meta() == r.Meta()) &&
-		(l.ShiftLeft() == r.ShiftLeft() || l.Shift() == r.Shift()) && (l.ShiftRight() == r.ShiftRight() || l.Shift() == r.Shift())
+	configured, incoming := h.Modifiers, v.Modifiers
```

Comment:
Not a show stopper, but why is this "almost sensor related logic" in the LDK part of this? Just worth asking as most of our "business logic" is in Olive Helps, not here.

Reply:
I think putting this logic here does make sense, since the consumer shouldn't have to be checking what keys were pressed, they just want to know if it's a match